lib/workers/schedule_worker/schedules_builder.rb

```diff
@@ -0,0 +1,41 @@
+class ScheduleWorker < WorkerBase
+  class SchedulesBuilder
+    include Vmdb::Logging
+
+    attr_reader :schedules
+
+    def initialize(scheduler, queue, schedule_category = nil)
+      @scheduler = scheduler
+      @queue = queue
```

Comment:
@imtayadeway And of course, the .new call would no longer accept the `@queue` and the builder wouldn't touch it.

### before
````ruby
builder = SchedulesBuilder.new(@system_scheduler, @queue, :schedules_for_all_roles)
```

### after
```ruby
builder = SchedulesBuilder.new(@system_scheduler, :schedules_for_all_roles)
```